umm-lib/src/cmr/umm/iso_smap/granule/spatial.clj

```diff
@@ -1,20 +1,30 @@
 (ns cmr.umm.iso-smap.granule.spatial
   "Contains functions for parsing and generating the ISO SMAP granule spatial"
-  (:require [cmr.common.xml :as cx]
-            [cmr.umm.umm-granule :as g]
-            [cmr.umm.iso-smap.spatial :as spatial]))
+  (:require
+   [cmr.common.xml :as cx]
+   [cmr.common.util :as util]
+   [cmr.umm.umm-granule :as g]
+   [cmr.umm.iso-smap.spatial :as spatial]))
 
 (def extent-path
   [:composedOf :DS_DataSet :has :MI_Metadata :identificationInfo :MD_DataIdentification
    :extent :EX_Extent :geographicElement])
 
+(defn- orbit?
+  "Returns true or false depending on the existence of the :ascending-crossing key existing in the
+  spatial coverage map."
+  [spatial-map]
+  (some #(= :ascending-crossing %) (keys spatial-map)))
```

Comment:
I think `(some #{:ascending-crossing} (keys spatial-map))` is the idiomatic way to check for the existence of a key.